corehq/apps/reports/filters/forms.py

```diff
@@ -45,8 +45,7 @@ def display_lang(self):
         """
             This should return the lang code of the language being used to view this form.
         """
-        # todo make this functional
-        return 'en'
+        return self.request.couch_user.language or 'en'
```

Comment:
Good point, @czue. couch_user can be an instance of CouchUser, InvalidUser and PublicUser. InvalidUser and PublicUser both extend CouchUser, which has a language property. So it should always be there, even if it isn't set.